src/test/resources/org/jabref/logic/importer/fileformat/AutosavedSharedDatabase.bib

```diff
@@ -1,6 +0,0 @@
-% DBID: 13ceoc8dm42f5g1iitao3dj2ap
```

Comment:
Please do not delete any existing test bib files.

Reply:
We just followed this instruction: https://twitter.com/JabRef_org/status/1209951516116865024
Anyway, I restored the files now.